reference/tools/user-guide.md

```diff
@@ -6,184 +6,270 @@ aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incre
 
 # TiDB Ecosystem Tools User Guide
 
-Currently, TiDB has multiple ecosystem tools. Some of them have overlapping functionality, and some are different versions of the same tool. This document introduces each of these tools, illustrates their relationship, and describes when to use which tool for each TiDB version.
+The TiDB ecosystem has a wealth of tools for data migration, backup & restore for users with different use cases to choose from. 
 
-## TiDB ecosystem tools overview
+- Some of the functionalities of these tools might overlap. For example, TiDB Loader, TiDB Lightning and TiDB DM can all do full data loading. 
+- Some of the tools might have evolved. For example, TiDB Binlog will be evolved to CDC (Change Data Capture). 
+- Some of the tools are designed to support specific TiDB versions and the others might be deprecated as user requirements change.
 
-TiDB ecosystem tools can be divided into:
+This guide is specifically designed to help you better understand these tools and therefore make an informed decision while choosing these tools to support your business.
 
-- Data import tools, including full import tools, backup and restore tools, incremental import tools, and so forth.
-- Data export tools, including full export tools. incremental export tools, and so forth.
+## Data import (restore or data replication)
 
-The two types of tools are discussed in detail below.
+### Full data import tools
 
-### Data import tools
+#### TiDB Lightning
 
-#### Full import tool TiDB Lightning
+[TiDB Lightning](/reference/tools/tidb-lightning/overview.md) (Lightning) is a tool used for the fast full import of large amounts of data into a TiDB cluster. Currently, TiDB Lightning supports reading SQL dump exported via Mydumper or CSV data source. 
 
-[TiDB Lightning](/reference/tools/tidb-lightning/overview.md) is a tool used for fast full import of data into a TiDB cluster.
+You can use it in the following two scenarios:
 
-> **Note:**
->
-> When you import data into TiDB using TiDB Lightning, there are two modes:
->
-> - The default mode: Use `tikv-importer` as the backend. In this mode, the cluster can not provide normal services during the data import process. It is used when you import large amounts (TBs) of data.
-> - The second mode: Use `TiDB` as the backend (similar to Loader). The import speed is slower than that in the default mode. However, the second mode supports online import.
+- Import large amounts of new data quickly
+- Backup and restore all the data
```

Comment:
lightning only can do import data, can't do back up

Reply:
I'll delete the above three lines.